### ISSUE TITLE
Exceptions flooding after the fresh installation

### ISSUE THREAD

**Issue by Hlibix** [uurha/BetterSceneManagement#22]
Also, when inspecting (actually) created Setting asset
```
UnityException: Creating asset at path Assets\Better\SceneManagement\Resources\SceneLoaderSettings.asset failed.
Better.EditorTools.SettingsTools.BetterSettingsTools`1[T].LoadOrCreateScriptableObject () (at ./Library/PackageCache/com.uurha.bettereditortools@0.9.2/Editor/SettingsTools/BetterSettingsTools.cs:45)
Better.SceneManagement.EditorAddons.SceneSettingsValidator.Validate (Better.SceneManagement.Runtime.SceneLoaderSettings& settings) (at ./Library/PackageCache/com.uurha.betterscenemanagement@0.2.1/Editor/EditorAddons/SceneSettingsValidator.cs:26)
Better.SceneManagement.EditorAddons.SceneSettingsValidator.ValidateSettings () (at ./Library/PackageCache/com.uurha.betterscenemanagement@0.2.1/Editor/EditorAddons/SceneSettingsValidator.cs:32)
Better.SceneManagement.EditorAddons.SceneSettingsValidator..cctor () (at ./Library/PackageCache/com.uurha.betterscenemanagement@0.2.1/Editor/EditorAddons/SceneSettingsValidator.cs:14)
Rethrow as TypeInitializationException: The type initializer for 'Better.SceneManagement.EditorAddons.SceneSettingsValidator' threw an exception.
Better.SceneManagement.EditorAddons.SceneLoaderSettingsEditor.OnInspectorGUI () (at ./Library/PackageCache/com.uurha.betterscenemanagement@0.2.1/Editor/EditorAddons/SceneLoaderSettingsEditor.cs:22)
UnityEditor.UIElements.InspectorElement+<>c__DisplayClass72_0.<CreateInspectorElementUsingIMGUI>b__0 () (at <ac87b54e58ee4be198261fd5c8030d52>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)
```

**Comment by Hlibix** [comment]
0.2.1

**Comment by uurha** [comment]
Resolved in version 0.2.3
